### PR TITLE
initialize php sdk in polling cron job

### PR DESCRIPTION
**Pull request by cwalv**
[ch24176]

### CODE REVIEW

File: Cron/Order.php [hunk 1]
```diff
@@ -79,8 +88,10 @@ public function __construct(
      */
     public function execute() : void
     {
+        $this->config->initSdkConfiguration();
```
Comment (CTDoriott) [inline Cron/Order.php:91]: @cwalv Thanks for catching this! I had this on my LightSail and didn't realize it wasn't in my commits. We may want to move this to before we declare the LoggingClient on line 81. That way the logging client object is aware of the config values that have been initialized.

File: Cron/Order.php [hunk 2]
```diff
@@ -63,11 +69,14 @@ class Order
     /**
      * Default constructor
      *
+     * @param Config $config
      * @param Order $order
```
Comment (davehensleyns8) [inline Cron/Order.php:73]: I know this isn't your change, but we should probably take this opportunity to make the second `@param` in the docblock match what's actually getting passed to the constructor.

File: Cron/Order.php (hunk 1 above)
Comment (cwalv) [reply Cron/Order.php]: good catch; fixed

File: Cron/Order.php (hunk 2 above)
Comment (cwalv) [reply Cron/Order.php]: done 👍

Comment (davehensleyns8) [reply Cron/Order.php]: Thanks! But the variable name needs to be updated too.

**Pull request merged**